Assets/Scripts/Waypoint Scripts/WaypointProperties.cs

```diff
@@ -113,6 +117,14 @@ void Update()
             UpdateGroundpointLine();
         }
 
+        private void ComputeWorldScaleActual() {
+            // TODO: check if already computed
+            WorldScaleCurrent = World.transform.localScale;
+            WorldScaleActual.x = WorldScaleCurrent.x / WorldScaleInitial.x;
+            WorldScaleActual.y = WorldScaleCurrent.y / WorldScaleInitial.y;
```

Comment:
didn't test it out so not sure how the math works exactly but seems like `WorldScaleActual` is used as a corrective factor in your waypoint gameObjects so it renders as a fixed visual size no matter the world scaling.

If that is the case, I think you want WorldScaleInitial / WorldScaleCurrent but not entirely sure

Reply:
the formula is from isaacs v1. the math seems to work as the wp coordinates are correctly updated (and the drone follows them) even when we scale the world